CONTRIBUTING.md

```diff
@@ -1,16 +1,27 @@
-# Making changes to nodejs.dev
+# Contributing Guide
+
+## 🚀 Get Started
+
+* Fork the nodejs.dev repo
+* Clone your fork `git clone git@github.com:<githubid>/nodejs.dev.git`
```

Comment:
This is only one of the three methods to clone the project. Plus there's now the option to use codespaces too where cloning locally is not required at all. #1745 was done using that only.

Any way we can mention those ways too?

Reply:
@manishprivet Whats the other way? Also, I read about codespaces but never actually used it so im not sure how that works.

Reply:
Added them in except for codespaces. I don’t see that tab when I try to clone.